test/e2e/run.test.ts

```diff
@@ -22,14 +24,16 @@ describe('edgekit basic run behaviour', () => {
   // look at jest-playwright.config.js
   const testUrl = 'http://localhost:9000';
 
+  const matchingVector = [1, 1, 1];
```

Comment:
@naripok do you think we could use this to create a rule: https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/naming-convention.md#enforce-that-all-const-variables-are-in-upper_case

Which allows UPPER_CASE only for hardcoded values?

Reply:
Dude, the rule as it is there enforces UPPER_CASE for all the const variables, not only for the hardcoded ones (I've tested it).
I'll get some research done, however, to see if I can find anything suiting our needs.